src/cascadia/TerminalConnection/ConhostConnection.h

```diff
@@ -37,6 +37,7 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         DWORD _outputThreadId;
         HANDLE _hOutputThread;
         PROCESS_INFORMATION _piConhost;
+        GUID _guid; // A "unique" session identifier for connected client

```

Comment:
It might be worthwhile to poke this through the `ITerminalConnection` interface as well, but that might mean we have to use `winrt::guid` instead

Reply:
feel free to wontfix this comment if it looks like scope creep. it's also something we can totally add later.

Reply:
Ah, I'm still learning the class structures.  I'll see if I can do that.  winrt is a completely unknown space to me.

Reply:
Managed it, I think, sending update shortly.